packages/fluentui/react-bindings/src/index.ts

```diff
@@ -15,6 +15,7 @@ export { default as unstable_useDispatchEffect } from './hooks/useDispatchEffect
 export { default as useIsomorphicLayoutEffect } from './hooks/useIsomorphicLayoutEffect';
 export { default as useStateManager } from './hooks/useStateManager';
 export { default as useStyles } from './hooks/useStyles';
+export { default as useUnhandledProps } from './hooks/useUnhandledProps';
```

Comment:
As we are exporting it, I believe it deserves changelog entry

Reply:
Agree, added